content/issues/1/_index.md

```diff
@@ -3,7 +3,7 @@ type: issue
 layout: single
 title: Issue One
 number: 1
-issue_theme: Transformations
+theme: Transformations
```

Comment:
I *think* that @gwijthoff and I used `issue_theme` instead of `theme` here to avoid confusion with Hugo theme settings, but I don't know if that actually matters in the context where we use this.

Reply:
as far as I could tell there were no repercussions. I think `theme` is stored in `.Site.Theme` whereas this would be `.Params.theme` or something, but if it causes conflict elsewhere we can switch it back.